### PR TITLE
Migrate Settings Left Menu Tab To Newer Material#2120

### LINKED ISSUE #2120: Migrate Settings Left Menu Tab To Newer Material
**Actual Behaviour**

Currently, older material ui is used for left menu panel

**Expected Behaviour**

Migrate it

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2120 

Changes: 
- Migrated Left menu tab, dropdown to material UI
- Use a dynamic way of creating repeatable menu items.

With this, material UI for chat susi is completed.

Demo Link: https://pr-2120-1-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd surge problem, please see again its working

**Comment by fragm3** [comment]
@akshatnitd I will proceed with Redux integration for settings once this is merged

File: src/components/ChatApp/Settings/Settings.react.js [hunk 1]
```diff
@@ -906,6 +919,47 @@ class Settings extends Component {
...
+          />
+        </React.Fragment>
+      );
+    });
+  };
+
+  generateDropDownMenu = () => {
+    return menuObj.map(obj => {
+      return (
+        <MenuItem value={obj.name} className="setting-item">
```
Comment (akshatnitd) [inline src/components/ChatApp/Settings/Settings.react.js]: Add a key to the returned element.

**Comment by fragm3** [comment]
@akshatnitd made the changes

**Pull request merged**